Gemfile

```diff
@@ -2,23 +2,18 @@
 
 source "https://rubygems.org"
 
-ruby RUBY_VERSION
-
 gemspec
 
-DECIDIM_VERSION = ">= 0.17.0"
```

Comment:
why remove?

Reply:
Why it's required here? It's in gemspec, isn't?